services/brig/src/Brig/Provider/DB.hs

```diff
@@ -406,10 +406,16 @@ paginateServiceTags tags start size = liftClient $ do
     let start' = Name (maybe "" (toLower . fromName) start)
     let size'  = size + 1
     let tags'  = unpackTags tags
-    p <- queryAll start' size' tags'
+    p <- filterPrefix (fromName start') <$> queryAll start' size' tags'
     r <- mapConcurrently resolveRow (result p)
     return $! ServiceProfilePage (hasMore p) (catMaybes r)
   where
+    filterPrefix :: Text -> Page TagRow -> Page TagRow
+    filterPrefix prefix p = do
+        let prefixed = filter (\(Name n, _, _) -> prefix `isPrefixOf` (toLower n)) (result p)
```

Comment:
is prefix always lower case here?

Reply:
Yes, it's lowered [here](https://github.com/wireapp/wire-server/pull/158/files#diff-1c9f610afacc03a5973a7a38e5ac370fR406)